azurerm/resource_arm_autoscale_setting_test.go

```diff
@@ -819,35 +902,17 @@ resource "azurerm_subnet" "test" {
   address_prefix       = "10.0.2.0/24"
 }
 
-resource "azurerm_storage_account" "test" {
-  name                     = "accsa%s"
-  resource_group_name      = "${azurerm_resource_group.test.name}"
-  location                 = "${azurerm_resource_group.test.location}"
-  account_tier             = "Standard"
-  account_replication_type = "LRS"
-
-  tags {
-    environment = "staging"
-  }
-}
-
-resource "azurerm_storage_container" "test" {
-  name                  = "vhds"
-  resource_group_name   = "${azurerm_resource_group.test.name}"
-  storage_account_name  = "${azurerm_storage_account.test.name}"
-  container_access_type = "private"
-}
-
 resource "azurerm_virtual_machine_scale_set" "test" {
-  name                = "acctvmss-%d"
-  location            = "${azurerm_resource_group.test.location}"
-  resource_group_name = "${azurerm_resource_group.test.name}"
-  upgrade_policy_mode = "Manual"
+  name                   = "acctvmss-%d"
+  location               = "${azurerm_resource_group.test.location}"
+  resource_group_name    = "${azurerm_resource_group.test.name}"
+  upgrade_policy_mode    = "Automatic"
+  single_placement_group = "false"
 
   sku {
-    name     = "Standard_F2"
+    name     = "Standard_DS1_v2"
     tier     = "Standard"
-    capacity = 2
+    capacity = 30
```

Comment:
needs confirmation, but I believe this can go down to 1

Reply:
It should also be able to goto 0 as well.